docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:24.04 AS aos-core-build-base
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV PATH="/root/.local/bin:${PATH}"
+
+# Install basic build dependencies
```

Comment:
Add empty line after comment as the comment is related to group of operators/commands.

Reply:
done